sources/packages/api/src/route-controllers/student/student.controller.ts

```diff
@@ -146,6 +149,27 @@ export class StudentController extends BaseController {
     );
   }
 
+  /**
+   * This returns only a part of the EducationProgram details to fetch
+   * for the of Student
+   * @param programId
+   * @returns
+   */
+  @Get("/education-program/:programId/summary")
+  async get(
```

Comment:
can you rename the function

Reply:
the complete route looks like api/students/education-program/44/summary. I feel its explanatory, saying the education-program summary for a student.

Reply:
the is student controller, IMO, If the function name is `get` , then router should be `api/student/:id`

Reply:
the id of the student is not used in the query, its a plain query to fetch the education program description for any student irrespective. As its for the student, I have moved it to the student controller.

Reply:
Let me rephrase, IMO, it makes sense to give a function name as `get`, when the router is `api/student/:id`, here the case is different right?.  `get` is not an appropriate name for the function, in this scenario IMO. thoughts @andrewsignori-aot  @dheepak-aot

Reply:
Ho you meant the get :). I get it now 👍 Thanks will rename